src/CodeStyle/Core/Tests/Microsoft.CodeAnalysis.CodeStyle.LegacyTestFramework.UnitTestUtilities.csproj

```diff
@@ -1,40 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Odd that git doesn't see this as a rename.

Reply:
I was surprised as well.